app/views/active_admin/resource/_page_compound_body_component_form.html.arb

```diff
@@ -0,0 +1,255 @@
+            a 'USWDS margin units',
+              href: uswds_margin_link,
+              class: 'usa-link usa-link--external',
+              target: '_blank'
+            span "Ex: '3' would equate to 'margin-bottom-3'. Defaults to 0."
+          end
+        end
+
+        # Margin top
+        li class: 'select input margin-top-container',
+           id: "page_page_components_attributes_#{placeholder}_component_attributes_margin_top_input" do
+          label 'Margin top', for: "page_page_components_attributes_#{placeholder}_component_attributes_margin_top", class: 'label'
+          select value: component&.margin_top || nil,
+                 id: "page_page_components_attributes_#{placeholder}_component_attributes_margin_top",
+                 name: "page[page_components_attributes][#{placeholder}][component_attributes][margin_top]" do
+            uswds_margin_units.each do |unit|
+              option "#{unit}", selected: component&.margin_top === unit
+            end
+          end
+
+          para class: 'inline-hints' do
+            span "Choose the top margin for this 'Compound Body' component (option are based on USWDS units:"
+            a 'USWDS margin units',
+              href: uswds_margin_link,
+              class: 'usa-link usa-link--external',
+              target: '_blank'
+            span "Ex: '3' would equate to 'margin-top-3'. Defaults to 0."
+          end
+        end
+
+        page_component_images = component&.page_component&.page_component_images
+
+        # Text alignment
+        li class: 'select input text-alignment-container',
+           id: "page_page_components_attributes_#{placeholder}_component_attributes_text_alignment_input",
+           style: "#{'padding-bottom: 36px' if page_component_images.blank?}" do
+          label 'Text alignment', for: "page_page_components_attributes_#{placeholder}_component_attributes_text_alignment", class: 'label'
+          select value: component&.text_alignment || nil,
+                 id: "page_page_components_attributes_#{placeholder}_component_attributes_text_alignment",
+                 name: "page[page_components_attributes][#{placeholder}][component_attributes][text_alignment]" do
+            option 'Left', selected: component&.text_alignment === 'Left' || component&.text_alignment.blank?
+            option 'Right', selected: component&.text_alignment === 'Right'
+          end
+          para "Choose the text alignment for this 'Compound Body' component (left or right). Defaults to left.",
+               class: 'inline-hints'
+        end
+
+        # Page component images header bar
+        div class: "page-component-images-header display-flex flex-align-center #{'display-none' if page_component_images.blank?}" do
+          span 'Image list'
+        end
+
+        # PageComponentImage list
+        ol id: "page_page_components_attributes_#{placeholder}_page_component_images_ol", class: 'page-component-images-ol' do
```

Comment:
One thing I was unable to figure out how to do was to make the `PageComponentImage` list a partial so we could use it for other components in the future. If you have any examples of how to do this, feel free to let me know and I can make the necessary changes.